app/controllers/application_controller.rb

```diff
@@ -10,7 +10,8 @@ def switch_locale &action
 
   def extract_locale_from_subdomain
     parsed_locale = request.subdomains.first
-    I18n.available_locales.map(&:to_s).include?(parsed_locale) ? parsed_locale : nil
+    I18n.available_locales.map(&:to_s).include?
```

Comment:
nếu line quá dài mà xuống dòng thì chuyển sang dùng if .. else thay vì dùng toán tử 3 ngôi